lib/ecstatic.js

```diff
@@ -124,13 +124,26 @@ var ecstatic = module.exports = function (dir, options) {
               url: parsed.pathname + '.' + defaultExt + ((parsed.search)? parsed.search:'')
             }, res, next);
           }
-          else {
-            // Try to serve default ./404.html
+          else if (!handleError) {
+            // If we're not handling errors/fallbacks, bail out now
+            middleware({
+              url: req.url,
+              statusCode: 404
+            }, res, next);
+          }
+          else if (path.basename(parsed.pathname, defaultExt) === '200.') {
```

Comment:
Also: Can we put this behind a flag? I don't know what to call it, but this should be opt-in behavior, at least for now.